.github/workflows/featured_issues.yml

```diff
@@ -22,13 +22,17 @@ jobs:
 
           # Save the issues to a featured-issues.json
           echo "$issues" | jq 'map({number, title, url, author, assignees, createdAt, state, comments_count: .comments | length})' > packages/frontendmu-data/data/featured-issues.json
+          echo "export default " > packages/frontendmu-data/data/featured-issues.ts
+          cat packages/frontendmu-data/data/featured-issues.json | jq '.' >> packages/frontendmu-data/data/featured-issues.ts
```

Comment:
@k3ii @n-d-r-d-g 
This looks nice, but probably not maintainable at all. Could we add comments to explain what's going on here? I'm thinking of the scenario when we come back to this 8 months later and don't feel confident to touch it

Reply:
Definitely not maintainable!
@k3ii will create a separate bash script to perform the conversion. @MrSunshyne You can leave this PR for now I guess.

Reply:
Do you want me add comments? @MrSunshyne